openstack/er/v3/associations/results.go

```diff
@@ -0,0 +1,109 @@
+package associations
+
+import (
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+// SingleResp is the structure that represents the association details and the request information of the API request.
+type SingleResp struct {
```

Comment:
```suggestion
type CreateResp struct {
```

we can also do not export the struct

Reply:
CreateResp is used.